### ISSUE TITLE
WorldPainter having issues generating world

### ISSUE THREAD

**Issue by a-111** [MattiBorchers/MinecraftEarthMap#23]
My WorldPainter doesn't seem to be able to load Cities40k.png while generating the world. Can't find the issue in the script. Anyone else having this issue or have a fix for it?

**Comment by Creeper553** [comment]
That picture needs to be downloaded from the link in the source code?

**Comment by MattiBorchers** [comment]
Yes, you need to download ALL images and change the "path" variable in the script.
